From 2a3c3e619fd1480ef875d5af856ebbd91d3bac50 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Mon, 28 Dec 2015 14:11:26 +0800
Subject: [PATCH 0810/5242] MLK-12081 ARM: dts: imx6qp-sabresd: remove the
 power on gpio

commit  a6bd81ecbda13d0b87ce69bfc9b902779617e72f from
https://source.codeaurora.org/external/imx/linux-imx.git

In regulator implementation, the gpio power on is used
as the fixed regulator, it is already manipulated by
the regulator driver, and always enabled.
Remove the power on gpio in pcie dts

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qp-sabresd.dts |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index 61691ddd..559d2d5 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -117,7 +117,6 @@
 };
 
 &pcie {
-	power-on-gpio = <&gpio3 19 0>;
 	reset-gpio = <&gpio7 12 0>;
 	status = "okay";
 };
-- 
1.7.9.5

